src/vm/docs/mod.rs

```diff
@@ -361,39 +361,60 @@ const NOT_API: SimpleFunctionAPI = SimpleFunctionAPI {
 const GEQ_API: SimpleFunctionAPI = SimpleFunctionAPI {
     name: Some(">= (greater than or equal)"),
     signature: "(>= i1 i2)",
-    description: "Compares two integers, returning `true` if `i1` is greater than or equal to `i2` and `false` otherwise.",
-    example: "(>= 1 1) ;; Returns true
+    description: "Compares two integers, returning `true` if `i1` is greater than or equal to `i2` and `false` otherwise.
+i1 and i2 must be of the same type. In Clarity1, the `>=`-comparable types are `int` and `uint`. In Clarity2 (from Stacks 2.1),
+the `>=`-comparable types are expanded to include `string-ascii`, `string-utf8` and `buff`.
+    ",
+    example: r#"(>= 1 1) ;; Returns true
 (>= 5 2) ;; Returns true
-"
+(>= "baa" "aaa") ;; Returns true
+(>= 0x02 0x01) ;; Returns true
+(>= 5 u2) ;; Throws type error
+"#
 };
 
 const LEQ_API: SimpleFunctionAPI = SimpleFunctionAPI {
     name: Some("<= (less than or equal)"),
     signature: "(<= i1 i2)",
-    description: "Compares two integers, returning true if `i1` is less than or equal to `i2` and `false` otherwise.",
-    example: "(<= 1 1) ;; Returns true
+    description: "Compares two integers, returning true if `i1` is less than or equal to `i2` and `false` otherwise.
+i1 and i2 must be of the same type. In Clarity1, the `<=`-comparable types are `int` and `uint`. In Clarity2 (from Stacks 2.1),
```

Comment:
More of a question for the group -- is it `Clarity1` and `Clarity2`, or is it `Clarity 1` and `Clarity 2`, or is it something else?